ui/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@xstate/react": "^1.6.2",
+    "classnames": "^2.3.1",
```

Comment:
This is just a tiny dependency to make chaining CSS classes or conditionally applying CSS classes a bit easier!